packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -172,13 +222,60 @@ export class StarkDropdownComponent extends AbstractStarkUiComponent implements
 		if (changes["multiSelect"]) {
 			this.isMultiSelectEnabled = StarkComponentUtil.isInputEnabled(this.multiSelect);
 		}
+
+		if (changes["isDisabled"]) {
+			if (typeof this.dropdownFormControl !== "undefined") {
+				this.logger.warn(reactiveWarningMessage);
+			}
+
+			if (typeof this.formControl !== "undefined") {
+				if (this.isDisabled) {
+					this.formControl.disable();
+				} else {
+					this.formControl.enable();
+				}
+			}
+		}
+
+		if (changes["required"]) {
+			this.setDefaultBlank();
+			if (
+				!changes["required"].firstChange &&
+				typeof this.dropdownFormControl === "undefined" &&
+				typeof this.formControl !== "undefined"
+			) {
+				if (this.required) {
+					this.formControl.setValidators([Validators.required]);
+				} else {
+					this.formControl.clearValidators();
+				}
+
+				this.formControl.updateValueAndValidity();
+			}
+		}
+
+		if (changes["value"]) {
+			if (typeof this.dropdownFormControl !== "undefined") {
+				this.logger.warn(reactiveWarningMessage);
+			}
+
+			if (
+				!changes["value"].firstChange &&
+				typeof this.value !== "undefined" &&
+				typeof this.formControl !== "undefined" &&
+				((Array.isArray(this.value) && this.formControl.value + "" !== this.value.toString()) ||
```

Comment:
Just to confim... is this working fine? The `toString()` and the `string + ""` give the same results? does it work in IE and Firefox? Cause I have the feeling that this really depends on the Javascript engine of the browser...

Reply:
I solved this issue by using isEqual method from lodash 😊